notebooks/notebooks/paper/katago-new-ckpt-evals.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #93.    handles, labels = list(by_label.values()), list(by_label.keys())
</pre><p>what's the purpose of putting <code>handles, labels</code> into a dict and then extracting them back out? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Haha idk what was happening here. My guess I just relied on copilot too much :)</p>